lib/elixir/test/elixir/code_test.exs

```diff
@@ -55,13 +55,45 @@ defmodule CodeTest do
                end)
     end
 
-    test "includes column information when possible" do
+    test "includes column information on unused variables" do
       assert {_, [%{position: {1, 12}}]} =
                Code.with_diagnostics(fn ->
                  quoted = Code.string_to_quoted!("if true do var = :foo end", columns: true)
                  Code.eval_quoted(quoted, [])
                end)
     end
+
+    test "includes column information on unused aliases" do
+      sample = """
+      defmodule FooAlias.Bar do end
+      defmodule FooAlias do 
+        alias FooAlias.Bar
+      end
```

Comment:
Instead of defining a new module, use any of the modules in Elixir stdlib. Such as `alias URI` or `alias String.Chars`.